README.md

```diff
@@ -39,26 +39,30 @@ Typical Lava-DL workflow:
 We highly recommend cloning the repository and using pybuilder to setup lava.
  You will need to install pybuilder for the same.
 
-**Note:** We assume you have already setup Lava with virtual environment.
-[TODO:verify on Windows and MacOS]
+**Note:** We assume you have already setup Lava with virtual environment. Make sure PYTHONPATH contains path to Lava core library first.
```

Comment:
... within a virtual ...
... make sure the PYTHONPATH contains the path to the Lava...